content/en/security/cloud_security_management/setup/without_infrastructure_monitoring.md

```diff
@@ -2,26 +2,32 @@
 title: Setting Up CSM without Infrastructure Monitoring
 ---
 
-You can set up CSM without infrastructure monitoring, so you can save on costs.
+In addition to setting up CSM with or without an Agent, you can also set it up without Infrastructure Monitoring.
 
-**Tip**: In your CSM settings, set up [resource evaluation filters][1] to limit the number of hosts you need security on.
+<!-- ## Set up CSM on your AWS account -->
 
-## Set up CSM on your AWS account
-
-1. Go to the [AWS Integration configuration page][2] in Datadog.
+1. Navigate to the [AWS Integration configuration page][2] in Datadog.
 1. On the **Configuration** tab, select the account you want to enable CSM on.
-1. To turn off infrastructure monitoring, under the selected account number, on the **Metric Collection** tab, click the **disable metric collection** link. Then, click **Disable Metric Collection** to confirm.
+
+   If you don't see the required account, add it by clicking **Add AWS Account(s)** and following the onscreen prompts.
+1. To turn off infrastructure monitoring on a given account, under the selected account number, on the **Metric Collection** tab, click the **disable metric collection** link. Then, click **Disable Metric Collection** to confirm.
 1. To turn on CSM, on the **Resource Collection** tab, turn on the **Enable Resource Collection** toggle, and select the **Enable Cloud Security Management** checkbox.
 1. Click **Save**.
 
-## Set up CSM on your Azure subscription
+**Note**: In your CSM settings, set up [resource evaluation filters][1] to limit the number of hosts you need security on.
+
+<!-- ## Set up CSM on your Azure subscription
```

Comment:
I haven't seen us commit commented-out content to prod before, so just wanted to flag that. I don't see why it hurts anything and I wouldn't expect it to get picked up in search -- it might be something other people do and I just haven't encountered it!